charts/fission-all/templates/buildermgr/deployment.yaml

```diff
@@ -5,15 +5,18 @@ metadata:
   labels:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     svc: buildermgr
+    part-of: fission
```

Comment:
I would rather prefer `application` instead of `part-of`. I will @sanketsudake review the PR. We should also see if value should be figured out from chart or release.

Reply:
`application`  label is already used in the some of the fission component deployments.
We need to use some different label to achieve this

Reply:
if fission is already setting the labels in the core components. Do we really need this PR?

Reply:
IMO if you have installed Fission with helm chart, typically there would be release name labels etc. 
OR chart label.
```
chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

I am not ok with PR overall since it solves a specific case and doesn't help in broad perspective.

One way I can consider this is by allowing custom labels/annotations for Fission users via helm chart values, where users can use them for any use-case or purpose they are targeting.